app/javascript/dashboard/store/modules/conversations/getters.js

```diff
@@ -13,6 +16,12 @@ const getters = {
     const comparator = {
       latest: (a, b) => b.last_activity_at - a.last_activity_at,
       sort_on_created_at: (a, b) => a.created_at - b.created_at,
+      sort_on_priority: (a, b) => {
+        return (
+          CONVERSATION_PRIORITY_ORDER[a.priority] -
+          CONVERSATION_PRIORITY_ORDER[b.priority]
+        );
+      },
```

Comment:
Why the need to add this here? Do we not get it from the server?

Reply:
@scmmishra If we are adding any new conversation to the state via action cable events, we should also include the necessary sorting in the getter.

Reply:
Ahh, makes sense!